build.gradle.kts

```diff
@@ -5,7 +5,7 @@ buildscript {
 }
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id("com.android.application") version "8.3.1" apply false
+    id("com.android.application") version "8.2.2" apply false
```

Comment:
if version 8.2.2 does not work for you, you might want to just update your Android Studio IDE

Reply:
@aprilslis what is wrong with version 8.3.1 ?